swap_meet/vendor.py

```diff
@@ -1,2 +1,64 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory = None) :
+        if not inventory:
+            self.inventory = []
+        else:
+            self.inventory = inventory
```

Comment:
Since you have a guard clause on line 3, you don't need the dangling `else` on line 5. 
You could write it like:
```python
if not inventory:
    self.inventory = []

self.inventory = inventory
```

You might also see this written as a ternary:

`my_var = "some value" if some_condition else "other value"`

In this situation, that would look like

```python
self.inventory = [] if inventory is None else inventory